lineman/app/components/navbar/navbar.haml

```diff
@@ -7,9 +7,9 @@
     .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
       %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
         %i.fa.fa-lg.fa-inbox>
+          %span.badge{ng-show: 'unreadThreadCount() > 0'}
```

Comment:
If you want to sort that out you need to remove the .btn class. It's not
really BEM to mix both utility classes and BEM classes. Making a button
mixin would be the way to go.

On Sun, 9 Aug 2015 at 08:53 Aaron Thornton <notifications@github.com> wrote:

> In lineman/app/components/navbar/navbar.haml
> <https://github.com/loomio/loomio/pull/2384#discussion_r36581748>:
>
> > @@ -7,9 +7,9 @@
> >      .lmo-navbar__item{ng-class: '{"lmo-navbar__item--selected": selected == "inboxPage"}'}
> >        %a.btn.lmo-navbar__btn.lmo-navbar__btn-icon.has-badge{href: '/inbox', title: "{{ 'navbar.unread_title' | translate }}", tabindex: 2}
> >          %i.fa.fa-lg.fa-inbox>
> > +          %span.badge{ng-show: 'unreadThreadCount() > 0'}
>
> Sorry just a frustrated dig at all the class names on one element after
> refactoring my dropdown arrow PR.
>
> —
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2384/files#r36581748>.
>